### ISSUE TITLE
Some posts aren't correctly assigned to a language

### ISSUE THREAD

**Issue by Aran-Fey** [polylang/polylang#287]
Unfortunately I don't know how this happened, so I can't tell you how to reproduce it. But somehow Polylang has messed up associating languages with the posts I've created.

As you can see in this screenshot, I have a total of 36 posts in 3 languages:

![beitragsliste](https://user-images.githubusercontent.com/22036272/43888156-1fe86a6e-9bc1-11e8-9dcf-90f029686b8a.png)

But the stats displayed by Polylang don't match (17 + 13 + 12 != 36):

![languages](https://user-images.githubusercontent.com/22036272/43888071-e0cafffe-9bc0-11e8-88d8-b09e26977c89.png)

Furthermore, some of my posts don't show up on my blog front page because they're not correctly associated with a language. On [the german (default language) front page](https://user-images.githubusercontent.com/22036272/43888430-f0e20b7a-9bc1-11e8-8206-3570f903a933.png), the posts show up as expected. But on [the english front page](https://user-images.githubusercontent.com/22036272/43888437-f40b6ff8-9bc1-11e8-9f9d-aa34cc76636f.png), some posts are missing, even though they [clearly have an english translation](https://user-images.githubusercontent.com/22036272/43888442-f666b834-9bc1-11e8-9c47-cef57ec29782.png).

I tried to re-assign the language of those posts, (i.e. change "english" to "german" and then back to "english"), but that didn't help. The only "solution" I found was to delete the post entirely and re-create it from scratch.

**Comment by Chouby** [comment]
The number of posts displayed in the Languages list counts all postd of all post types, not only the post type 'post'. 

As for the other issue, please provide sufficient information to reproduce it ( also the template that you did not fill includes several informations that would have help). We can only investigate and possibly fix if you can provide us a clear description with steps to reproduce.

**Comment by Aran-Fey** [comment]
Sorry, not sure which part of the template you're referring to. I have already admitted that I don't know how to reproduce the issue, and I wouldn't have opened it if I had found a duplicate.

What may not have been clear is that I've already "fixed" the problem by re-creating the problematic posts, so I cannot test if the issue still exists with the latest master branch version or with a default WordPress theme. I'm afraid I can't provide any more information than I already have. (But yes, I understand it's near-impossible for you to debug.)
